app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -133,6 +134,19 @@ public void setZulipActivity(ZulipActivity zulipActivity) {
         this.zulipActivity = zulipActivity;
     }
 
+    //store narrowFilter before switching theme
+    private NarrowFilter narrowFilter = null;
```

Comment:
So we won't need to assign null additionally
`private NarrowFilter narrowFilter;`